docs/guides/security-component-migration.md

```diff
@@ -60,7 +60,7 @@ v2 will receive weekly releases occurring every Tuesday morning.
 | `DelimitedList`                    | View changes [here](#delimitedlist)                                                                               | `@carbon/ibm-products` | Low    |
 | `Dropdown`                         | [Carbon v11 Migration guide][111]                                                                                 | `@carbon/react`        | Low    |
 | `DropdownSkeleton`                 | The deprecated prop `inline` has been removed                                                                     | `@carbon/react`        | Low    |
-| `ErrorPage`                        | **Deprecated:** [View replacement][4]                                                                             | `@carbon/ibm-products` | High   |
+| `ErrorPage`                        | **Deprecated:** [~~View replacement~~][4] [View replacement][9]                                                   | `@carbon/ibm-products` | High   |
```

Comment:
This results in two links, one that is struck through. Is this what was intended?

<img width="439" alt="image" src="https://github.com/carbon-design-system/ibm-products/assets/909118/fb90ca08-990a-4243-b507-1444596218d5">

Reply:
on this line, i noticed `HTTPErrors` being shown as a replacement for `ErrorPage`. but as we are deprecating `HTTPErrors` now, and replacing it with `FullPageErrors`, i struck it out, but i think we can remove it completely. will do it.